specification/security/resource-manager/Microsoft.Security/preview/2023-02-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1494 @@
+          "type": "object",
+          "description": "Azure Resource Manager metadata containing createdBy and modifiedBy information.",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Security connector data",
+          "$ref": "#/definitions/SecurityConnectorProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "SecurityConnectorProperties": {
+      "type": "object",
+      "description": "A set of properties that defines the security connector configuration.",
+      "properties": {
+        "hierarchyIdentifier": {
+          "type": "string",
+          "description": "The multi cloud resource identifier (account id in case of AWS connector, project number in case of GCP connector)."
+        },
+        "hierarchyIdentifierTrialEndDate": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The date on which the trial period will end, if applicable. Trial period exists for 30 days after upgrading to payed offerings."
+        },
+        "environmentName": {
+          "type": "string",
+          "description": "The multi cloud resource's cloud name.",
+          "enum": [
+            "Azure",
+            "AWS",
+            "GCP",
+            "Github",
+            "AzureDevOps",
+            "GitLab"
+          ],
+          "x-ms-enum": {
+            "name": "cloudName",
+            "modelAsString": true
+          }
+        },
+        "offerings": {
+          "type": "array",
+          "description": "A collection of offerings for the security connector.",
+          "items": {
+            "$ref": "#/definitions/cloudOffering"
+          },
+          "x-ms-identifiers": []
+        },
+        "environmentData": {
+          "type": "object",
+          "description": "The security connector environment data.",
+          "$ref": "#/definitions/EnvironmentData"
+        }
+      }
+    },
+    "EnvironmentData": {
+      "type": "object",
+      "description": "The security connector environment data.",
+      "discriminator": "environmentType",
+      "required": [
+        "environmentType"
+      ],
+      "properties": {
+        "environmentType": {
+          "type": "string",
+          "description": "The type of the environment data.",
+          "enum": [
+            "AwsAccount",
+            "GcpProject",
+            "GithubScope",
+            "AzureDevOpsScope",
+            "GitlabScope"
+          ],
+          "x-ms-enum": {
+            "name": "environmentType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "AwsEnvironmentData": {
+      "type": "object",
+      "description": "The aws connector environment data",
```

Comment:
>aws [](http://example.com/codeflow?start=26&length=3)

AWS should be capitalized everywhere in descriptions I expect

Reply:
capitalized all